bubblewrap.c

```diff
@@ -1315,6 +1440,31 @@ parse_args_recurse (int    *argcp,
           op->source = argv[1];
           op->dest = argv[2];
 
+          argv += 2;
+          argc -= 2;
+        }
+      else if (strcmp (arg, "--overlay") == 0)
```

Comment:
This option should be rejected when privileged, to avoid handing out abilities that the kernel thinks should only be given to root.